app/assets/javascripts/discourse/app/components/user-info.js

```diff
@@ -1,34 +1,40 @@
 import Component from "@ember/component";
 import { alias } from "@ember/object/computed";
+import {
+  attributeBindings,
+  classNameBindings,
+} from "@ember-decorators/component";
 import { prioritizeNameInUx } from "discourse/lib/settings";
 import { userPath } from "discourse/lib/url";
 import discourseComputed from "discourse-common/utils/decorators";
 
-export default Component.extend({
-  classNameBindings: [":user-info", "size"],
-  attributeBindings: ["data-username"],
-  size: "small",
-  "data-username": alias("user.username"),
-  includeLink: true,
-  includeAvatar: true,
+@classNameBindings(":user-info", "size")
+@attributeBindings("dataUsername:data-username")
+export default class UserInfo extends Component {
+  size = "small";
+
+  @alias("user.username") dataUsername;
+
+  includeLink = true;
+  includeAvatar = true;
```

Comment:
I'm not suggesting moving any stuff around 😝

Reply:
sorted